### PR TITLE
Monitor exits

### PR DESCRIPTION
**Pull request by potuz**
This PR adds logs for voluntary exits to the validator monitor. It is based on #9898

### CODE REVIEW

File: beacon-chain/monitor/service.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+var (
+	log = logrus.WithField("prefix", "monitor")
+)
+
+// ValidatorMonitorConfig contains the list of validator indices that the
+// monitor service tracks, as well as the event feed notifier that the
+// monitor needs to subscribe.
+type ValidatorMonitorConfig struct {
+	TrackedValidators []types.ValidatorIndex
```
Comment (rauljordan) [inline beacon-chain/monitor/service.go:16]: Why not a map? That way you can get rid of the for loop over the entire slice every time you call TrackedIndex

File: beacon-chain/monitor/process_block.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+// processSlashings logs the event of one of our tracked validators was slashed
+func (s *Service) processSlashings(blk block.BeaconBlock) {
+	for _, slashing := range blk.Body().ProposerSlashings() {
+		idx := slashing.Header_1.Header.ProposerIndex
+		if s.TrackedIndex(idx) {
+			log.WithFields(logrus.Fields{
+				"ProposerIndex":  idx,
+				"SlashedInSlot:": blk.Slot(),
+				"Slot":           slashing.Header_1.Header.Slot,
```
Comment (terencechain) [inline beacon-chain/monitor/process_block.go:21]:
```suggestion
				"HeadSlot":           slashing.Header_1.Header.Slot,
```
then add a `CurrentSlot`

File: beacon-chain/monitor/process_exit.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import (
+	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/block"
+	"github.com/sirupsen/logrus"
+)
+
+// processExitsFromBlock logs the event of one of our tracked validators' exit was
+// included in a block
+func (s *Service) processExitsFromBlock(blk block.BeaconBlock) {
```
Comment (terencechain) [inline beacon-chain/monitor/process_exit.go:11]: I think i will like cleaner abstraction. By renaming this function to `processExits` and just take in `exits []*ethpb.SignedVoluntaryExit` as argument

File: beacon-chain/monitor/service.go (hunk 1 above)
Comment (terencechain) [reply beacon-chain/monitor/service.go:16]: +1

File: beacon-chain/monitor/process_exit.go (hunk 3 above)
Comment (potuz) [reply beacon-chain/monitor/process_exit.go:11]: The problem is that we need to report in which block this was included, so either we take also the slot, or we take the block as I am taking here.

Comment (terencechain) [reply beacon-chain/monitor/process_exit.go:11]: Ah. Block is fine then

**Review by terencechain — APPROVED** [review]

**Review by prestonvanloon — APPROVED** [review]
Approved. Please fix merge conflict

**Review by prestonvanloon — APPROVED** [review]
I fixed merge conflict and re-ran gazelle.

**Pull request merged**